proprietary/tokens/src/components/amsterdam/link.tokens.json

```diff
@@ -0,0 +1,94 @@
+{
+  "amsterdam": {
+    "link": {
+      "color": { "value": "{amsterdam.color.primary-blue}" },
+      "icon": {
+        "size": { "value": "1.2em" }
+      },
+      "text-decoration": { "value": "none" },
+      "text-underline-offset": { "value": "9px" },
+      "text-decoration-thickness": { "value": "2px" },
+      "text-decoration-skip-ink": { "value": "none" },
+      "font-family": {
+        "value": "{amsterdam.typography.font-family}"
+      },
+      "font-size": {
+        "narrow": { "value": "{amsterdam.typography.text-level.6.font-size.narrow}" },
+        "wide": { "value": "{amsterdam.typography.text-level.6.font-size.wide}" }
+      },
+      "font-weight": {
+        "value": "{amsterdam.typography.font-weight.normal}"
+      },
+      "focus": {
+        "color": { "value": "{amsterdam.color.dark-blue}" },
+        "text-decoration": { "value": "underline" }
```

Comment:
[We should add `focus-offset` here](https://amsterdam.github.io/design-system/?path=/docs/docs-designrichtlijnen-focus-indicator--docs)